src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj

```diff
@@ -67,7 +67,7 @@
       <HintPath>..\..\..\packages\Microsoft.Azure.KeyVault.Core.1.0.0\lib\net40\Microsoft.Azure.KeyVault.Core.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.Azure.Management.Authorization">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Authorization.2.0.0\lib\net40\Microsoft.Azure.Management.Authorization.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Authorization.2.6.0-stack\lib\net40\Microsoft.Azure.Management.Authorization.dll</HintPath>
```

Comment:
What is this? You added '-stack' to your package?  This is absolutely not allowed.  Also, what is the relationship with the previous version?  We are deleting this package, and you need to resubmit a PR that uses a real version number.